linux/LinuxProcessList.c

```diff
@@ -1473,8 +1474,8 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
       {
          bool prev = proc->usesDeletedLib;
 
-         if ((ss->flags & PROCESS_FLAG_LINUX_LRS_FIX) ||
-             (settings->highlightDeletedExe && !proc->procExeDeleted && !proc->isKernelThread && !proc->isUserlandThread)) {
+         if (((ss->flags & PROCESS_FLAG_LINUX_LRS_FIX) || (settings->highlightDeletedExe && !proc->procExeDeleted))
```

Comment:
Maybe move the now top-level conditions `!proc->isKernelThread && !proc->isUserlandThread` to the front for improved readability?

Reply:
Done and rebased.